docs/conf.py

```diff
@@ -47,8 +47,6 @@
     "sphinx.ext.viewcode",
     "sphinx.ext.napoleon",
     "sphinx.ext.mathjax",
-    "recommonmark",
```

Comment:
Why is recommonmark taken out? Need this when mixing MD and rst

Reply:
re-added.